docs/troubleshooting.md

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting 
```

Comment:
This is troubleshooting building from source, so I'm not sure that it is right to split it off from the building-from-source instructions.  Thoughts @evildandelions ?